src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
+                + getArea() + " sq. units, width: "
+                + width + " units, height: "
+                + height + " units, color: " + getColor());
```

Comment:
Avoid using 'System.out.println' directly in the 'draw' method. Consider using a dedicated method to return the string representation of the object and then print it elsewhere in your application.